package.json

```diff
@@ -40,18 +42,23 @@
     "eslint-plugin-react": "^7.7.0",
     "fs-extra": "^5.0.0",
     "husky": "^0.14.3",
-    "hyperapp": "^1.2.5",
+    "hyperapp": "^1.2.6",
     "jest": "^22.4.3",
     "prettier": "^1.11.1",
     "rollup": "^0.57.1",
     "rollup-plugin-babel": "4.0.0-beta.2",
     "rollup-plugin-commonjs": "^9.1.0",
     "rollup-plugin-node-resolve": "^3.3.0",
-    "rollup-plugin-uglify": "^3.0.0"
+    "rollup-plugin-uglify": "^3.0.0",
+    "typescript": "^2.9.2"
+  },
+  "peerDependencies": {
+    "@types/node": ">=6.x",
+    "hyperapp": "^1.2.6"
   },
   "scripts": {
     "lint": "eslint benchmark src test tools",
-    "test": "jest",
+    "test": "jest --coverage --no-cache && tsc -p test/ts",
```

Comment:
Why do we need `--no-cache` here?

Reply:
I'm not sure. https://github.com/facebook/jest/issues/6610